src/accel/cuda/GPUSolver.cu

```diff
@@ -1118,27 +1155,31 @@ void GPUSolver::initializeFSRs() {
     FP_PRECISION* host_FSR_volumes = _FSR_volumes;
     int* host_FSR_materials = _FSR_materials;
 
-    /* Allocate memory on device for FSR volumes and Material indices */
-    cudaMalloc((void**)&_FSR_volumes, _num_FSRs * sizeof(FP_PRECISION));
-    cudaMalloc((void**)&_FSR_materials, _num_FSRs * sizeof(int));
+    cudaMalloc(&_FSR_volumes, _num_FSRs * sizeof(FP_PRECISION));
+    getLastCudaError();
+    cudaMalloc(&_FSR_materials, _num_FSRs * sizeof(int));
+    getLastCudaError();
```

Comment:
Hm, I think it would be good to have them. Does any compiler macro variable get defined when --debug is used? Then I could only enable them with ifdef DEBUG or the like.

Reply:
It doesnt. We'll have to think about this. Current abstraction is using those log_printf(DEBUG)